x-pack/plugins/alerts/server/alerts_client/alerts_client.ts

```diff
@@ -248,6 +268,15 @@ export class AlertsClient {
         error: null,
       },
     };
+
+    this.auditLogger?.log(
+      alertRuleEvent({
+        action: AlertRuleAction.CREATE,
+        outcome: EventOutcome.UNKNOWN,
```

Comment:
Is this "unknown" because we haven't actually tried to create yet? If so, shouldn't this be explicit about not having an _unknown outcome_ as much as just not having reached an outcome yet?

I might be misunderstanding the reasons it's unknown 🤔 
Just wondering if 'Unknown' is conceptually the right way of thinking about this event as it could be confusing if we see this in the log...

Reply:
Fair enough, I guess the very existence of an unknown outcomes suggests to me that there _was_ an outcome, we just couldn't identify it...
I can go either way, which is why I was happy to approve 👍